### PR TITLE
Add Dockerfile and info how to run Mephisto from docker

### PR DESCRIPTION
**Pull request by maci0**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  ./make.sh install;\  
+  cd ~; git clone https://github.com/reswitched/Mephisto.git; \
+  cd Mephisto;\
+  pip install -r requirements.txt;\
+  make
+
+EXPOSE 24689
+
+
+ENTRYPOINT ["/root/Mephisto/ctu"]
```
Comment (misson20000) [inline Dockerfile]: I'm not familiar with Docker. Does this mean we're building as root? Is this normal for Docker?

File: README.md [hunk 2]
```diff
@@ -68,3 +68,24 @@ Alternatively, you can pass a single NSO file on the command line:
...
+First build the docker image, this may take some time
+
+```bash
+docker build -t reswitched/mephisto .
+```
+
+To run Mephisto it needs access to your NSO/NRO files, make sure to bind mount the location into the container.
+
+```bash
+docker run -ti --rm -p 24689:24689 -v /home/$USER:/home/$USER reswitched/mephisto --load-nro /$HOME/Coding/libtransistor/build/test/test_helloworld.nro
```
Comment (misson20000) [inline README.md]: Is there any reason we're not consistent between `-v /home/$USER:/home/$USER` and `--load-nro /$HOME/...`, how we use `/home/$USER` in one place and `$HOME` in the other? We probably also shouldn't make presumptions about where a user has libtransistor installed.

**Review by misson20000 — CHANGES_REQUESTED** [review]
Looks mostly good to me. I'm not familiar with Docker so I'll have to trust that this all works the way one might expect. After addressing the two things I commented on, this will probably be good to merge.

File: Dockerfile (hunk 1 above)
Comment (maci0) [reply Dockerfile]: unless a different user is specified this is normal. I can easily change this to use a different user, the nobody user for example. but if your system has usernamespaces enabled root maps to a different user anyway.

will change it nonetheless.

File: README.md (hunk 2 above)
Comment (maci0) [reply README.md]: uhm, no reason, will change it.

regarding libtransistor location, its just an example

**Pull request merged**